KC_Liberation_Master_Framework/karmakut/deployable_field_hospitals/server/init.sqf

```diff
@@ -8,10 +8,12 @@ karma_deployableFieldHospitals_server_deployFieldHospital = {
     if (isNull _player) exitWith {};
 
     private _playerUID = getPlayerUID _player;
-    if (!isNull (karma_deployableFieldHospitals_server_fieldHospitalsByPlayerUID getOrDefault [_playerUID, objNull])) exitWith {
+    private _existingFieldHospital = karma_deployableFieldHospitals_server_fieldHospitalsByPlayerUID getOrDefault [_playerUID, objNull];
+    if (!isNull _existingFieldHospital) exitWith {
         [
             karma_deployableFieldHospitals_shared_statusCodes_fieldHospitalAlreadyDeployed,
-            _playerUID
+            _playerUID,
+            _existingFieldHospital
```

Comment:
Passing this back to the client will ensure that JIP players that placed a field hospital in a previous session will rehydrate their state upon attempting to place a field hospital after having already placed one.

Reply:
wonderful